apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -84,7 +84,7 @@ const ProgressBubbleSet = React.createClass({
                 <ProgressPill
                   url={level.url}
                   status={level.status}
-                  text={i18n.unpluggedActivity()}
+                  text={level.isGoBeyond ? i18n.goBeyond() : i18n.unpluggedActivity()}
```

Comment:
I wonder if this could be something like `level.title || i18n.unpluggedActivity()` (essentially allow one to arbitrarily override the text of unplugged activites)